test/integration/auth.spec.ts

```diff
@@ -37,6 +37,8 @@ chai.use(chaiAsPromised);
 
 const expect = chai.expect;
 
+const authEmulatorHost = process.env.FIREBASE_AUTH_EMULATOR_HOST;
```

Comment:
We primarily run integration tests at release time to verify e2e functionality. Without the build/CI infrastructure to run these tests in the emulator mode, they will rarely get invoked and therefore are not all that useful. We need to also update the CI infrastructure to run these tests regularly (like how you set up the Python build pipeline for RTDB+emulator).

Reply:
As discussed, I'll make a follow up PR to enable testing against Emulator Suite since there are some unrelated tests that will only pass after the next Firebase CLI release. As an upside, we will have coverage for tests not just for Auth, but also all of Firestore ones and most RTDB ones.

In the meantime, I ran these locally with the unreleased CLI patches and they pass.